docs/getting_started.md

```diff
@@ -0,0 +1,114 @@
+
+```console
+kubectl apply -f https://raw.githubusercontent.com/cert-manager/istio-csr/main/docs/example-issuer.yaml
+```
+
+## Export the Root CA to a Local File
+
+While it's possible to configure Istio such that it can automatically "discover" the root CA, this can be dangerous in
+some specific scenarios involving other security vulnerabilities, enabling [signer hijacking attacks](https://github.com/cert-manager/istio-csr/issues/103#issuecomment-923882792).
+
+As such, we'll export our Root CA and configure Istio later using that static cert.
+
+
+```console
+# Export our cert from the secret it's stored in, and base64 decode to get the PEM data.
+kubectl get -n istio-system secret istio-ca -ogo-template='{{index .data "tls.crt"}}' | base64 -d > ca.pem
+
+# Out of interest, we can check out what our CA looks like
+openssl x509 -in ca.pem -noout -text
+
+# Add our CA to a secret
+kubectl create secret generic -n cert-manager istio-root-ca --from-file=ca.pem=ca.pem
+```
+
+## Installing istio-csr
+
+istio-csr is best installed via Helm, and it should be simple and quick to install. There
+are a bunch of other configuration options for the helm chart, which you can check out [here](https://github.com/cert-manager/istio-csr/blob/main/deploy/charts/istio-csr/README.md).
+
+```console
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+
+# We set a few helm template values so we can point at our static root CA
+helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+	--set "app.tls.rootCAFile=/var/run/secrets/istio-csr/ca.pem" \
+	--set "volumeMounts[0].name=root-ca" \
+	--set "volumeMounts[0].mountPath=/var/run/secrets/istio-csr" \
+	--set "volumes[0].name=root-ca" \
+	--set "volumes[0].secret.secretName=istio-root-ca"
+
+# Check to see that the istio-csr pod is running and ready
+kubectl get pods -n cert-manager
+NAME                                       READY   STATUS    RESTARTS   AGE
+cert-manager-aaaaaaaaaa-11111              1/1     Running   0          9m46s
+cert-manager-cainjector-aaaaaaaaaa-22222   1/1     Running   0          9m46s
+cert-manager-istio-csr-bbbbbbbbbb-00000    1/1     Running   0          63s
+cert-manager-webhook-aaaaaaaaa-33333       1/1     Running   0          9m46s
+```
+
+## Installing Istio
+
+If you're not running on kind, you may need to do some additional [setup tasks](https://istio.io/latest/docs/setup/platform-setup/) before installing Istio.
+
+This istio installation configuration is required for:
+
+- The CA server to be disabled in istiod
+- Ensuring Istio workloads request certificates from istio-csr
+- Ensuring that the istiod certificates and keys are mounted from the Certificate created when installing istio-csr
```

Comment:
I was a bit lost while reading here: what is the "installation configuration"?


```suggestion
We will be using the `istioctl` CLI to install Istio. In order for istio-csr to work, we will pass a custom IstioOperator manifest to `istioctl`. The custom manifest does the following:

- It disables the CA server in istiod,
- It makes sure that Istio workloads request certificates from istio-csr,
- It ensures that the istiod certificates and keys are mounted from the Certificate created when installing istio-csr.

Let us first download the custom IstioOperator manifest, and then use istioctl to install Istio:
```

Reply:
Much better wording yeah, thanks :+1: